rai_core_flask/requirements.txt

```diff
@@ -1,4 +1,4 @@
-Flask
+Flask>=3.0.0
```

Comment:
we should already be getting the latest flask version since there is no pin here - so the question is what is downgrading flask instead of getting the latest version?